src/test/java/carracing/CarRacingTest.java

```diff
@@ -0,0 +1,33 @@
+package carracing;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class CarRacingTest {
+    private CarRacing carRacing;
+    private List<String> carNames;
+    private List<Car> cars;
+
+    @BeforeEach
+    void setUp() {
+        cars = new ArrayList<>();
+        carNames = Arrays.asList("Acar", "Bcar", "Ccar");
+        for(int i = 0; i < carNames.size(); i++) {
+            cars.add(new Car(carNames.get(i)));
+        }
+        carRacing = new CarRacing(cars);
```

Comment:
다소 복잡하게 `List<Car> cars`를 만들어주고 있는 것 같아요. 테스트를 위한 환경 조성을 위해 `cars`를 초기화하는 것이 목적이라면 아래와 같이 간단하게 진행해볼 수도 있을 것 같아요 :) 
이는 `CarRacing` 객체가 `List<Car> cars`를 관리하기 때문에, 그 이전 단계인 `List<Car> cars의 생성은 무사히 되었다!` 라고 가정하고 시작해도 된다고 생각해요 :)
```suggestion
        cars = Arrays.asList(new Car("Acar"), new Car("Bcar"), new Car("Ccar"));
        carRacing = new CarRacing(cars);
```